pkgs/applications/networking/avalanchego/default.nix

```diff
@@ -0,0 +1,52 @@
+{ IOKit
+, buildGoModule
+, fetchFromGitHub
+, fetchpatch
+, lib
+, stdenv
+}:
+
+buildGoModule rec {
+  pname = "avalanchego";
+  version = "1.9.5";
+
+  src = fetchFromGitHub {
+    owner = "ava-labs";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-Uim2Q3LvNEV0C5/gFdAiXKy2oKbhhc/ifRyz5BIh6zE=";
+  };
+
+  # patch zondax/hid dependency manually on darwin
```

Comment:
If we don't patch conditionally on darwin, rewrite the comment here why we need the patch.

E.g. manually update zondax/hid dependency to fix build.

Reply:
@urandom2 And add a comment with the pr https://github.com/ava-labs/avalanchego/pull/1194 so that we don't forget to remove the patch again when it is merged or the dependency is updated in an other way.

Reply:
I thought that having the url in fetchpatch would be enough, plus the patch will fail to apply after it merges?

Reply:
You are right. That is already enough😃.